joomla/joomla-cms
# [4.0] Remove Confirm Password from user edit view

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/23345 .

### Summary of Changes
Confirm Password field removed from user edit in backend
### Testing Instructions
code review or test whether user edit forks as expected and Confirm Password is not present.
### Expected result
No Confirm Password field and user edit works.
### Actual result
Confirm Password field exists and user edit works
### Documentation Changes Required
no
